Modules/Microsoft365DSC/Modules/M365DSCDRGUtil.psm1

```diff
@@ -400,7 +400,7 @@ function Get-M365DSCDRGComplexTypeToString
                 $currentValue = $ComplexObject[$key]
                 if ($currentValue.GetType().Name -eq 'String')
                 {
-                    $currentValue = $ComplexObject[$key].Replace("'", "''").Replace("�", "''")
+                    $currentValue = $ComplexObject[$key].Replace("'", "''").Replace(" ", "''")
```

Comment:
Why this change?

Reply:
I didn't explicitly modify this line -- this might have been an artifact from VSCode -- that can be ignored.

Reply:
Any chance you can remove the change? I can't merge otherwise, you control the code for the PR. Thanks

Reply:
I've removed the change in the latest commit